changelogs/fragments/1695-parted-updatedregex.yaml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - parted - change the regex that decodes the partition size to better support different formats that parted uses.
+    Change the regex that validate parted's version string.
```

```suggestion
    Change the regex that validate parted's version string
```
Ah, just one detail: